packages/web_ui/src/components/SlavesPage.jsx

```diff
@@ -1,15 +1,27 @@
-import React, { useContext } from "react";
+import React, {useContext} from "react";
```

Comment:
There's no linting rule for this, but every other import in the project has spaces inside the `{ brackets }`. Also this is a random unrelated change.